module-router.c

```diff
@@ -730,13 +730,21 @@ static void set_pa_volume(pa_cvolume* destchannelVolume, int num_channels, uint3
  * @return pa_hook_result: The result of the hook function.
  */
 static pa_hook_result_t hook_callback_sink_input_put(pa_core *c, pa_sink_input *sink_input, struct userdata *u) {
-    ROUTER_FUNCTION_ENTRY;
+    pa_log_info("hook_callback_sink_input_put index=%d",sink_input->index);
     pa_assert(c);
     pa_assert(u);
     pa_assert(sink_input);
     bool corked = false;
     pa_sink_input_state_t state;
 
+    char* media_name = (char*) pa_proplist_gets(sink_input->proplist,PA_PROP_MEDIA_NAME);
+
+    if((media_name != NULL) && (0==strcmp(media_name,"pulsesink probe")))
+    {
+    	pa_log_info("hook_callback_sink_input_put probe found");
+    	return PA_HOOK_OK;
+    }
```

Comment:
This code is similar to hook_callback_sink_input_new, but little bit different.
Can you make it functionalized by same code? or can it be same?